clients/fides-js/src/lib/overlay.module.css

```diff
@@ -193,16 +200,21 @@ div#fides-consent-modal {
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
+}
 
-  .modal-header {
```

Comment:
Technically I'm adding `postcss` here: https://github.com/ethyca/fides/pull/3431

...which'll let us enable lots of CSS preprocessing features like nesting and other super-reasonable things like that. But let's do vanilla CSS for now!